bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -250,38 +256,59 @@ public void updateEnabledState(boolean enabled) {
         elevMin.setEnabled(e);
         elevMax.setEnabled(e);
     }
-}
 
+	private class GBC extends GridBagConstraints {
```

Comment:
There are SO many of these GBCs. I was thinking about trying to dig them out and see how similar they are. We really should have a common implementation of this given that I think a lot of it is copy-paste.

I don't mind doing that and it would be a good improvement, but I think I'd rather do it in another PR.